src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -465,14 +465,15 @@ InferenceProfiler::Create(
     std::unique_ptr<InferenceProfiler>* profiler,
     uint64_t measurement_request_count, MeasurementMode measurement_mode,
     std::shared_ptr<MPIDriver> mpi_driver, const uint64_t metrics_interval_ms,
-    const bool should_collect_metrics, const double overhead_pct_threshold)
+    const bool should_collect_metrics, const double overhead_pct_threshold,
+    const std::string& profile_export_file)
```

Comment:
I'm not convinced that this should be plumbed down into inference_profiler. If you look at how the csv output is handled, it is only passed into an object up at the perf_analyzer.cc level.  I can imagine another new object being created there.  Ideally the inference profiler knows nothing about this new option.

Either this story shouldn't plumb at all and the follow-on story should do so, or we need to wait on this story until we know where it is needed.

Reply:
Its unclear where the class that handles the raw data output will live just yet: perf_analyzer or inference_profiler. Given that, Tim is probably right to not plumb this down just yet. We can pass this option along after we know where that class lives.

Reply:
Good catch. Removed propagation of path to InferenceProfiler 👍